server/service/user/user.go

```diff
@@ -0,0 +1,85 @@
+package service
+
+import (
+	pb "github.com/PavelDonchenko/bookstorejRPC/server/gen/proto"
+	model "github.com/PavelDonchenko/bookstorejRPC/server/models"
+	repository "github.com/PavelDonchenko/bookstorejRPC/server/repository/user"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type UserService struct {
+	userRepo repository.UserRepository
+}
+
+func NewUserService(userRepo *repository.UserRepo) *UserService {
+	return &UserService{
+		userRepo: userRepo,
+	}
+}
+
+func (us *UserService) GetOne(id uint32) (*pb.GetUserResponse, error) {
+	user, err := us.userRepo.GetOne(id)
+
+	item := pb.UserItem{
+		Id:       uint32(user.ID),
+		Nickname: user.Nickname,
+		Email:    user.Email,
+		Password: user.Password,
+		CrateAt:  timestamppb.Now(),
+		UpdateAt: timestamppb.Now(),
+	}
+	return &pb.GetUserResponse{User: &item}, err
+}
+func (us *UserService) GetAll() (*pb.GetAllAUserResponse, error) {
+	users, err := us.userRepo.GetAll()
+
+	items := []*pb.UserItem{}
+
+	for _, v := range users {
+		u := &pb.UserItem{
+			Id:       uint32(v.ID),
+			Nickname: v.Nickname,
+			Email:    v.Email,
+			Password: v.Password,
+			CrateAt:  timestamppb.Now(),
```

Comment:
please update it according to our discussion

Reply:
fixed